build.cmd

```diff
@@ -204,6 +204,19 @@ call                                 "%__VSToolsRoot%\VsDevCmd.bat"
 
 REM =========================================================================================
 REM ===
+REM === Restore optimization profile data
+REM ===
+REM =========================================================================================
+
+echo %__MsgPrefix%Restoring the OptimizationData Package
```

Comment:
@janvorli PS thank you for cluing me in on why the FreeBSD CI job was failing too--it's unable to run managed code, just like you hinted. I'll make sure the `skiprestoreoptdata` is passed for now when building on FreeBSD.